test/inductor/test_fp8.py

```diff
@@ -34,7 +35,11 @@
 
 
 def _fix_fp8_dtype_for_rocm(
-    dtype: Union[torch.dtype, list[torch.dtype], tuple[torch.dtype]], device
+    dtype: Union[torch.dtype,
+    list[torch.dtype],
+    tuple[torch.dtype]],
```

Comment:
I think it reads better if the alternative types are all in one line; right now, they can accidentally read as arguments to the `_fix_fp8_dtype_for_rocm` method